examples/E2EBert/MLproject

```diff
@@ -6,15 +6,17 @@ entry_points:
   main:
     parameters:
       max_epochs: {type: int, default: 5}
+      train_num_samples: {type:int, default: 2000}
+      val_num_samples: {type:int, default: 200}
+      test_num_samples: {type:int, default: 200}
       gpus: {type: int, default: 0}
-      num_samples: {type: int, default: 15000}
-      vocab_file: {type: str, default: 'https://s3.amazonaws.com/models.huggingface.co/bert/bert-base-uncased-vocab.txt'}
```

Comment:
What is the reason for removing this config option?

Reply:
usually the users run this example in cpu. We have 5 epochs. it takes a very long time to complete the training with 5 epochs and 15000 samples . Hence, i reduced the count to 1000.